src/app/app-interactive-map/store/interactive-map.effects.ts

```diff
@@ -39,6 +40,18 @@ const preferredMap = (mapItems: MapItem[]): MapItem =>
 @Injectable()
 export class InteractiveMapEffects {
   @Effect()
+  fetchSpecies: Observable<Action> = this.actions$.pipe(
+    ofType(fromActions.FETCH_SPECIES),
+    switchMap(() => {
+      return this.http.get(`${environment.apis.warehouse}/organisms`);
+    }),
+    map((payload: {project_id: string, id: string, name: string, created: string, updated: string}[]) => new fromActions.SetSpecies(payload)),
+  );
+  @Effect()
+  setSpecies: Observable<Action> = this.actions$.pipe(
+    ofType(fromActions.SET_SPECIES),
+    map((action: fromActions.SetSpecies) => new SetSelectedSpecies(action.payload[0].id)));
```

Comment:
So this is where you could enforce preferred species - Similar to preferred maps https://github.com/DD-DeCaF/caffeine/blob/devel/src/app/app-interactive-map/store/interactive-map.effects.ts#L32-L37

I'm not sure about this yet, I'm open to suggestions. I was wondering if the backend should know the preference?

Reply:
I think it makes sense to make this decision in the frontend. An idea could f.ex. be to store last selected item per-browser and restore it on page load. And for first-time visitors fall back to the preference we define.